PlatformConfigurations/IEEE802.1xPortBasedAuthentication.md

```diff
@@ -0,0 +1,872 @@
+---
+title: IEEE 802.1x (Port Based Authentication)
+parent: Platform Configuration
+nav_order: 5
+layout: default
+---
+
+# IEEE 802.1x (Port Based Authentication)
+
+## Introduction
+
+In this guide, we will quickly explain what IEEE 802.1x (Port Based Authentication) is
+and give an example implementing it.
+
+IEEE 802.1x is a standard for port-based network access control that provides an authentication
+mechanism to devices wishing to join a LAN or WLAN. IEEE 802.1x uses the Extensible Authentication Protocol
+(EAP) to transmit authentication information over ethernet frameworks. The three main components
+include the **Supplicant, Authenticator, and Authentication Server**.
+
+- **Supplicant**: The Supplicant is the device attempting to access the network. It is often the one that
+  participates in the EAP transaction with the Authenticator.
+
+- **Authenticator**: The Authenticator is the network device that connects the supplicant to the network. It acts as the
+  access point for the Supplicant and controls the type of traffic flow between the Supplicant and
+  the network before and after authentication. The Authenticator sends
+  EAP requests to the Supplicant to identify a connecting device.
+  In IEEE 802.1x, this is also more broadly refered to as a Network Access Server.
+
+- **Authentication Server**: The Authentication Server receives requests for network access
+  and verifies devices attempting to connect to the network. This information is then passed back to the authenticator
+  where traffic flow is changed based on the results. This is also commonly refered to as a RADIUS Server.
+  It is also often known as an AAA (Authentication, Authorization, and Accounting) Server.
+
+The image below is a good visualization of the 802.1x Progression.
+
+![Seqeunce Diagram of the 802.1x Progression](../Images/ImagesForPlatformConfiguration/802.1x_Progression.png)
+
+In this document, we will be using the following free publicly available software:
+FreeRADIUS, hostapd, and wpa_supplicant to configure an example of IEEE 802.1x and MAC Address Authentication.
+
+## Example Configuration
+
+Consider the following topology:
+
+![IEEE-802.1x-Configuration](../Images/ImagesForPlatformConfiguration/IEEE-802.1x-ConfigurationExample.png)
+
+First, we will guide you through how to configure 802.1x and then
+show how to configure MAC Address Authentication
+
+## Installations
+
```

Comment:
where do you install this on a dent switch, on a server, on your laptop, ?????

Reply:
I have updated the **Installation** to clarify there are three separate devices and each utility should be installed on only one device.

![image](https://github.com/user-attachments/assets/35b06edc-451b-40c1-addd-e31b96b1a515)